src/transport/LogTransportType.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace LogHero\Client;
+
+
+abstract class LogTransportType {
+    const Async = 0;
```

Comment:
If its not needed for combining bitwise, i would recommend to use small descriptional strings instead. It's more helpful in case you need to display that information somewhere.

Reply:
Good point. Done